src/cli/aurora.rs

```diff
@@ -56,28 +56,27 @@ pub async fn execute_command(
         // Command::Benchmark
         Command::Read { subcommand } => match subcommand {
             ReadCommand::GetResult { tx_hash_hex } => {
-                let tx_hash =
-                    aurora_engine_types::H256::from_slice(&hex::decode(tx_hash_hex).unwrap());
+                let tx_hash = aurora_engine_types::H256(utils::hex_to_arr(&tx_hash_hex).unwrap());
```

Comment:
Now that we use `hex_to_arr` functions, it seems we can remove `unwrap()` with `?`. @birchmd can you remove unwrap as error handling PR is proceeding? https://github.com/aurora-is-near/aurora-cli-rs/pull/6.

Reply:
`aurora_engine_types::H256(utils::hex_to_arr(&tx_hash_hex)?);`

Reply:
Ok, so I guess we are in development stage? I was asked to put down request for changes, and I will do that in PR #6